im.pidgin.Pidgin3.yml

```diff
@@ -0,0 +1,128 @@
+app-id: im.pidgin.Pidgin3
```

Comment:
why not update https://github.com/flathub/im.pidgin.Pidgin to pidgin 3? Why does it need to be separate?

Reply:
They are not compatible and pidgin3 is still pre-alpha quality so they need to be installable side by side for people to try out pidgin 3 as right now it has only implemented a fraction of pidgin 2's functionality.